app/scripts/DataFetcher.js

```diff
@@ -197,7 +257,7 @@ export default class DataFetcher {
     const result = new Float32Array(numeratorData.length);
 
     for (let i = 0; i < result.length; i++) {
-      if (denominatorData[i] === 0.) result[i] = NaN;
+      if (denominatorData[i] === 0.0) result[i] = NaN;
```

Comment:
FYI, I think `=== 0` and `=== 0.0` are equivalent because JS doesn't distinguish between `int` and `float` (sadly)

Reply:
Linter complains about 0.